packages/application-components/src/components/modal-pages/internals/modal-page.tsx

```diff
@@ -110,6 +114,9 @@ const ModalPage = (props: Props) => {
     };
   }, [props.isOpen]);
   const { onClose } = props;
+
+  const TRANSITION_DURATION = stylesBySize[props.size].transitionTime;
```

Comment:
Hey Yahia, do we really need to pull the transition duration value here?
I guess we're using it now just in case the `ModalPage` component changes its size once rendered, right? (don't know such a use case).
If so, maybe we can just add the `props.size` to the dependencies array.

What do you think? 🤔

Reply:
Actually no, the transition value of `200` was used in the modal page already before my PR so I just replaced that value which was coming from the constant to the value coming from the stylesBySize object but still it was used already in 2 places. mostly when closing the modal to handle the closing animation and the timeout of the closing.

Reply:
Oh! I see.

Thanks for the explanation. 👍